server/profile.go

```diff
@@ -91,8 +91,87 @@ func GetProfile(w http.ResponseWriter, r *http.Request) {
 }
 
 func EditProfile(w http.ResponseWriter, r *http.Request) {
+	//Connect to datastore
+	ctx := context.Background()
+	dsClient, err := datastore.NewClient(ctx, Conf.Project)
+	if err != nil {
+		log.Println(err)
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte("datastore error"))
+		return
+	}
+
+	//Retrieve user from datastore
+	vars := mux.Vars(r)
+	userID := vars["id"]
+	if len(userID) == 0 {
+		w.WriteHeader(http.StatusOK)
```

Comment:
I would suggest here (and below) to return appropriate error codes unless there is a good reason to always return OK.